source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx

```diff
@@ -1,29 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <materialx version="1.38">
-  <!-- Implementation overrides for ESSL -->
+  <!--
+    TM & (c) 2021 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
```

Comment:
Not sure whether to put lucas films or Autodesk like in this file: resources/Materials/TestSuite/translation/standard_surface_to_UsdPreviewSurface.mtlx

Reply:
Good question, I copied it over from this file: libraries/stdlib/genglsl/stdlib_genglsl_impl.mtlx
@bernardkwok what should we do here?

Reply:
FYI: Removed the file.